### ISSUE TITLE
Tooltip: aria-describedBy not applied to target

### ISSUE THREAD

**Issue by dermartinschmitz** [primefaces/primeng#16630]
### Describe the bug

The tooltip documentation states that the tooltip's id should be used for the aria-describedby attribute of the target element. While the role and id are correctly assigned to the tooltip itself, the aria-describedby attribute is not automatically applied to the target element as expected.

### Environment

PrimeNG Version 17.18.11

### Angular version

18.2.0

### PrimeNG version

17.18.11

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20
